resources/helper.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# only for keys within [UNIT]
+function add_entry_to_service() {
+	SERVICE=/lib/systemd/system/$1
```

Comment:
There could be other locations of systemd service files, e.g. /etc/systemd/system, just to keep in mind. Should work for the purpose of a temporary workaround here.